app/views/cart/show.html.erb

```diff
@@ -30,7 +30,8 @@
       </td>
       <td>
         <center>
-          <p><%=quantity%></p>
+          <p id = "item-quantity-<%=item.id%>"><%=quantity%></p>
```

Comment:
Added an id to be able to test within the item's quantity section to make sure the numbers we test for are here and not elsewhere on the page